core/container.go

```diff
@@ -1322,6 +1312,29 @@ func (container *Container) export(
 	return res, nil
 }
 
+func (container *Container) Sha(ctx context.Context, gw bkgw.Client) (string, error) {
```

Comment:
IMO `Sha` is pretty unclear. I think it would be better to just call it `ImageRef` and have it return a ref like `my/repo@sha256:...`.

Reply:
fixed